carmajava/launch/cadillac_config/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+#  Copyright (C) 2018 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+version: '2'
+
+services:
+  carma:
+    image: usdot-carma/carma
+    network_mode: host
+    container_name: carma
+    volumes_from: 
+      - container:carma-config:ro
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
```

Comment:
Just a curiosity - I understand this location to be a ros convention (/home/user). Not sure, but I think this is mapping tha to /opt/carma/.ros.  Can we put the ros logs as well as the carma logs all under /opt/carma/logs just to make it easier to find everything?

Reply:
Yeah, we could map it that way, that's not a problem at all.

Reply:
For clarity's sake, mappings are written "HOST:CONTAINER" directories, so anything the container writes to `/home/carma/.ros` is written to `/opt/carma/.ros`. I could make a ros subfolder on the host at `/opt/carma/logs/ros`, and that way it's not hidden either if that's more desirable. Or we could just have it be `/opt/carma/logs/.ros`

I don't think mapping both of those volumes to host `/opt/carma/logs` will work, Docker might prevent double mapping like that or the container might be tripping over its own feet by doing so, either way it feels unlikely to work in my opinion.

Reply:
I think it would be nice to have everything bundled under /opt/carma/logs.  I can see not mapping two things to that exact directory, but yeah a subdir unter that called ros might be a good thing.  Seems like a mod to be considered as part of 135, not now.